lib/hanami/cli/commands/application.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "dry/files"
+
+require_relative "db/utils/database"
+
+module Hanami
+  module CLI
+    module Commands
+      class Application < Hanami::CLI::Command
+        module Environment
+          def call(**opts)
+            env = opts[:env]
+            ENV["HANAMI_ENV"] = env.to_s
+            super(**opts)
+          end
+        end
+
+        def self.inherited(klass)
+          super
+          klass.option(:env, required: false, default: "development", desc: "Application's environment")
+          klass.prepend(Environment)
+        end
+
+        def application
+          @application ||=
+            begin
+              require "hanami/init"
+              Hanami.application
+            end
+        end
+
+        def run_command(klass, *args)
+          klass.new(
+            out: out,
+            inflector: application.inflector,
+            fs: Dry::Files
```

Comment:
@solnic Shouldn't this be `Dry::Files.new`?

Reply:
@jodosha probably 🙂 it's not used yet (generators are disabled). I'll address this separately.